sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -413,6 +418,18 @@ public SentryId captureTransaction(
     return event;
   }
 
+  private void sortBreadcrumbsByDate(
+      final @NotNull SentryEvent event, final @NotNull Collection<Breadcrumb> breadcrumbs) {
+    final List<Breadcrumb> sortedBreadcrumbs = event.getBreadcrumbs();
```

Comment:
Here we modify `event#breadcrumbs`. Shouldn't we do instead:

```
final List<Breadcrumb> sortedBreadcrumbs = new ArrayList<>(event.getBreadcrumbs());?
```

Reply:
the outcome of this is a sorted breadcrumb in the event payload, so it's not needed to allocate a new ArrayList, I just got the reference to not call a getter a few times

Reply:
Ok. Do they need to be set then again in line 430?

Reply:
yep, good catch, it's a leftover of my previous implementation, gonna remove it, thx